languageserver/integration/integration.go

```diff
@@ -71,3 +74,64 @@ func NewFlowIntegration(s *server.Server, enableFlowClient bool) (*FlowIntegrati
 
 	return integration, nil
 }
+
+type FlowIntegration struct {
+	server *server.Server // todo remove since not used
+
+	entryPointInfo map[protocol.DocumentURI]*entryPointInfo
+	contractInfo   map[protocol.DocumentURI]*contractInfo
```

Comment:
Yes, I was thinking the same at first, but I think I didn't do it because there's only one that applies for a give URI, but then again, it might work with an interface. WIll look into this more, I anyhow miss some tests for codelenses.